pkg/codegen/python/doc.go

```diff
@@ -37,9 +37,18 @@ func (d DocLanguageHelper) GetDocLinkForPulumiType(pkg *schema.Package, typeName
 	return ""
 }
 
-// GetDocLinkForResourceType is not implemented at this time for Python.
+// GetDocLinkForResourceType returns the Python API doc for a type belonging to a resource provider.
 func (d DocLanguageHelper) GetDocLinkForResourceType(pkg *schema.Package, modName, typeName string) string {
-	return ""
+	var path string
```

Comment:
So the preview mostly looked good, except for an issue with Kubernetes. For example, take a look at this k8s [resource](content-bucket-3db671c.s3-website-us-east-1.amazonaws.com/docs/reference/pkg/kubernetes/admissionregistration/v1/mutatingwebhookconfiguration/?language=python) and click on the Python function. The raw module name is `admissionregistration.k8s.io/v1`, which should actually be transformed to `admissionregistration/v1`.

@lblackstone how does the current Python code gen for k8s handle module name transformations? Just wondering if we should keep it simple here and manually strip-off the common k8s domain names here (similar what we do [here](https://github.com/pulumi/pulumi/blob/master/pkg/codegen/docs/gen_kubernetes.go#L102)) or maybe use the Go module to package override before calling this method to pass the normalized module name?

Reply:
@praneetloke The k8s Python SDK hasn't been moved over to the schema codegen yet. You could either override the names manually, or set that as part of the language info in the k8s schema code.

Reply:
> or set that as part of the language info in the k8s schema code.

Yeah we would need to right? It will be needed for when the code gen is switched to be the schema-based one?

@sean1588 For now, I think we could manually replace the domain names here similar to what we do [here](https://github.com/pulumi/pulumi/blob/master/pkg/codegen/docs/gen_kubernetes.go#L102).

Reply:
> It will be needed for when the code gen is switched to be the schema-based one?

Yeah, I expect it will be needed soon, but the manual override is fine for now. Likely still several weeks out on Python being moved over to schema gen.

Reply:
Ok, thanks. Yeah I agree, we could just do it manually for the time being. I'll go ahead and just make the change to handle this case for now.

Reply:
Is there a tracking issue in the k8s repo? Would be good to add a comment referencing that issue.

Reply:
https://github.com/pulumi/pulumi-kubernetes/issues/1085